vulnerabilities/xss/index.js

```diff
@@ -43,16 +43,35 @@ module.exports = (function() {
 
 	api.post('/xss_post', function(req, res, next) {
 		res.set('X-XSS-Protection', '0'); // disable browser xss protection for chrome
-		var input = req.body.email;
+		var input = req.body.input;
 		var output = '<html>e-mail: ' + input + '</html>';
 		res.send(output);
 	});
 	api.post('/xss_post_safe', function(req, res, next) {
 		res.set('X-XSS-Protection', '0'); // disable browser xss protection for chrome
-		var input = escape(req.body.email);
+		var input = escape(req.body.input);
 		var output = '<html>e-mail: ' + input + '</html>';
 		res.send(output);
 	});
 
+	api.get('/header', function(req, res) {
```

Comment:
Comment that these are for assess mode sources, and something about that the Protect XSS rule does not look at headers ( for the most part ).